iOS Example Frame/iOS Example Frame/ViewController/HomeViewController.swift

```diff
@@ -27,10 +27,8 @@ class HomeViewController: UIViewController {
   override func viewDidLoad() {
     super.viewDidLoad()
 
-#if UITEST
```

Comment:
since this is a sample project and we'd need that both for UI tests and integration tests, I didn't see the need of such an optimisation here so removed it

Reply:
feel free to unresolve this if that doesn't sound okay